python/tvm/relay/frontend/common.py

```diff
@@ -548,6 +558,28 @@ def new_var(name_hint,
     return _expr.var(name_hint, type_annotation, shape, dtype)
 
 
+def check_tensor_array_shape(expr, dtype, mod):
```

Comment:
I think querying the fixed shape associated with a static tensor array would be very common thing to do. Does it make sense to have such utility function in prelude, rather than in frontend code?

Reply:
Yes. I'll move it to prelude.